pages/getting-started/installing-cypress.md

```diff
@@ -0,0 +1,334 @@
+npm install cypress --save-dev
+```
+
+Isto vai instalar o Cypress localmente como uma dependência de desenvolvimento em seu projeto.
+
+> Tenha certificado que você já tenha rodado `npm init` ou tenha o diretório `node_modules` ou o arquivo `package.json` no diretório raiz do seu projeto para garantir que o cypress foi instalado no diretório correto.
+
+> Note que o pacote npm do cypress é um wrapper que envolve o binário do Cypress. A versão do pacote npm determina qual versão do binário baixado. Como a versão 3.0, o binário baixado como um diretório de cache global para ser utilizado entre os projetos.
+
+> Boa prática
+
+> A abordagem recomendada é de instalar o Cypress usando o `npm` porque: 
+    - Cypress é versionada como as outras dependências.
+    - É muito simples rodar o Cypress na Integração Contínua.
+
+## `yarn add`
+
+Instalando o cypress usando o `yarn`:
+
+```
+cd /your/project/path
+yarn add cypress --dev
+```
+
+## download direto
+
+Se você não estiver utilizando Node ou npm no seu projeto ou você quer tentar usar o Cypress rapidamente, você sempre poderá realizar o download do Cypress diretamente do nosso CDN. 
+
+> Gravando as execuções para o Dashboard não é possível pelo download direto. Este download tem como oferecer uma forma rápida de utilizar o Cypress. Para gravar os testes no Dashboard, você precisará instalar o Cypress como uma depedência npm.
+
+O download direto sempre trará a última versão disponível. Sua plataforma será identificada automaticamente.
+
+Você pode descompactar manualmente e clicar duas vezes. Cypress irá ser executado sem precisar instalar nenhuma dependência.
+
+## Integração contínua
+
+Por favor leia nossa documentação sobre [Integração Contínua](#) para ajudar na instação do Cypress na CI (Integração Contínua). Quando estiver rodando linux você precisa'ra instalar algumas dependências de sistema ou você pode usar a nossa imagem docker que possui tudo que você precisa.
+
+# Abrindo cypress
+
+Se você usou `npm` para instalação, Cypress foi instalado no diretório `./node_modules`, com seus binários executáveis accessíveis em `./node_modules/.bin`.
+
+Agora você pode abrir o Cypress do seu projeto principal com uma das seguintes formas:
+
+A forma longa com o caminho completo
+```
+./node_modules/.bin/cypress open
+```
+
+Ou com o atalho usando `npm bin`
+```
+```
+
+Ou usando o `npx`
+
+nota: npx está incluido no npm > v5.2 ou pode ser instalado separadamente.
+```
+npx cypress open
+```
+
+Após um momento, o Cypress Test Runner será aberto.
+
+# Trocando navegadores
+
+O Cypress Test Runner tenta encontrar todos os browsers compatíveis na máquina do usuário. O drop down para selecionar um browser diferente está no topo lateral direito do Test Runner.
+
+![cypress test runner ui](https://docs.cypress.io/img/guides/browser-list-dropdown.80de3be3.png)
+
+Leia [Abrindo navegadores](#) para mais informações sobre como o Cypress controla um browser real durante os testes de ponta-a-ponta.
```

Comment:
```suggestion
Leia [Abrindo navegadores](#) para mais informações sobre como o Cypress controla um navegador real durante os testes de ponta-a-ponta.
```

I also suggest a `TODO` on the above line so that we remember to update the link when 'Abrindo navegadores' is translated.

Reply:
done